app/components/settings/profile-page/username-section.hbs

```diff
@@ -11,24 +11,27 @@
     {{#if @user.hasAnonymousModeEnabled}}
       <EmberTooltip @text="Your profile has anonymous mode enabled. Your GitHub username is not visible to others." />
     {{else}}
-      {{! activate this once implemented }}
-      {{! <EmberTooltip @text="This value is synced with your GitHub account. Click on the refresh button below to update your username." /> }}
-      <EmberTooltip @text="This value is synced with your GitHub account." />
+      <EmberTooltip @text="This value is synced with your GitHub account. Click on the refresh button below to update your username." />
     {{/if}}
   </div>
   <div class="text-gray-400 text-xs mt-1">
     {{#if @user.hasAnonymousModeEnabled}}
       Your profile has anonymous mode enabled. Your GitHub username is not visible to others.
     {{else}}
-      This value is synced with your GitHub account.
+      This value is synced with your GitHub account. Click on the refresh button below to update your username.
     {{/if}}
   </div>
 
-  {{!-- TODO: Bring this back once implemented }}
-  {{!-- <TertiaryButtonWithSpinner @size="extra-small" class="mt-4" @shouldShowSpinner={{false}}>
+  <TertiaryButtonWithSpinner
```

Comment:
If a user has anonymous mode enabled, should we show this?

Reply:
If a user has anonymous mode enabled and clicked on this, what would happen?

Reply:
makes sense. user shouldn't be able to change github username if anonymous